README.md

```diff
@@ -94,16 +111,19 @@ Add an ERD image to visualize the database schema.
 Reference the API documentation for details on how to interact with the API.
 
```

Comment:
Let's add the instruction for /api-docs route for the API documentation until we deploy the page.